### DISCUSSION TITLE
How to select edit branch besides main?

### DISCUSSION THREAD

**Discussion by henry2004y** [quarto-dev/quarto-web#582] [General]
Hi,

I would like to know if it is possible to set the git branch for the `edit this page` button in the html version? Currently it all assumes "main", but for example in my test repo "master" is used (see [here](https://henry2004y.github.io/keynotes_quarto/) and the edit button link error).

Thanks!

**Comment by cscheid** [comment]
If you go to quarto.org, hit the search button, and type "branch", you'll see the `repo-branch` option under Website Navigation:

<img width="1400" alt="image" src="https://user-images.githubusercontent.com/285675/214182223-6d67fa77-0731-4ce8-ae08-0c3ad90a151e.png">

Add that to your YAML.
